test/e2e/apibinding/apibinding_test.go

```diff
@@ -66,14 +68,19 @@ func TestAPIBinding(t *testing.T) {
 	consumer3Workspace := framework.NewWorkspaceFixture(t, server, orgClusterName)
 
 	cfg := server.DefaultConfig(t)
+	cfgCopy := rest.CopyConfig(cfg)
 
-	kcpClients, err := clientset.NewClusterForConfig(cfg)
+	cfg.Wrap(func(rt http.RoundTripper) http.RoundTripper {
+		return kcp.NewClusterRoundTripper(rt)
+	})
```

Comment:
`cfg.Wrap(kcp.NewClusterRoundTripper)`.

This has to become a central helper: `func NewClusterConfig(*rest.Config) *rest.Config`.